README.md

```diff
@@ -91,222 +90,32 @@ A tool that makes it easier to manage Akamai Sandboxes. Call the Sandbox API fro
 Options:
   -V, --version                                                 Output the current version.
   --debug                                                       Show debug information.
-  --edgerc <path>                                               Use edgerc file for authentication.
+  --edgerc <file>                                               Use edgerc file for authentication.
   --section <name>                                              Use this section in edgerc file that contains the credential set.
   --accountkey <account-id>                                     Use given internal parameter.
   -h, --help                                                    Output usage information.
 
 Commands:
   help [command]                                                Displays help information for the given command.
   install                                                       Downloads and installs the Sandbox Client software.
-  list|ls [options]                                             Lists sandboxes that you have managed locally.
-  show [sandbox-identifier]                                     Provides details about a sandbox.
-  rules [sandbox-identifier]                                    Shows rule tree for sandbox.
-  use <sandbox-identifier>                                      Sets the identified sandbox as currently active.
+  list|ls [options]                                             Lists sandboxes that are available locally.
+  show [sandbox-id]                                             Shows details about the sandbox and JWT expiration date.
+  rules [sandbox-id]                                            Shows a rules tree for the sandbox.
```

Comment:
`a rules` -> `rules`

Reply:
rules tree is a singular. 
so Shows a rules tree   OR Shows rules treeS